src/node_crypto.cc

```diff
@@ -741,6 +743,64 @@ void SecureContext::AddCRL(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void SecureContext::UseExtraCaCerts(const char* fname) {
+  extra_root_certs_file = fname;
+}
+
+static void emitWarning(Environment* env, const char* fmt, ...) {
+  char warning[1024];
+  va_list ap;
+
+  va_start(ap, fmt);
+  vsnprintf(warning, sizeof(warning), fmt, ap);
+  va_end(ap);
+
+#if 0
```

Comment:
`Call()` interacts poorly with domains and other mechanisms that hang off async_hooks so it's really only safe for internal JS code.  W.r.t. handle leaks, `Call` does that as well (the return value.)